es-ds-components/components/es-error-page.vue

```diff
@@ -4,7 +4,19 @@ import IconCommunitySolar from './icon/community-solar.vue';
 import IconInfoBlogPost from './icon/info-blog-post.vue';
 import IconMarketplace from './icon/marketplace.vue';
 
-const ES_DOMAIN = process.env.ES_DOMAIN || 'https://www.energysage.com';
+const props = defineProps({
+    errorType: {
+        type: String,
+        required: true,
+    },
+    esDomain: {
+        type: String,
+        default: null,
+    },
+});
+
+const ES_DOMAIN = props.esDomain || 'https://www.energysage.com';
```

Comment:
One thing I realized during testing - if I load the error page docs page locally, then pass in a value to the `esDomain` prop, the change is not reflected.

This is because the `ES_DOMAIN` variable and the `linkItems` variable that depends on it are not set up to be reactive to props changes. Now that we have a prop, everything that flows from that prop should generally always be reactive in a Vue component.

This is easy to do if we just wrap a `computed(() => { ... })` around the logic for both `ES_DOMAIN` (which at that point can probably become a camelCase name since it's no longer a fixed constant) and `linkItems`.

Reply:
I made those changes and I still see the correct domain when I pass `es-domain` as a test in `error-page.vue`.